src/components/pages/search.js

```diff
@@ -79,6 +84,11 @@ function Search({ initialState }) {
       title: t('updateDate'),
       dataIndex: 'updatedOn',
       render: renderUpdatedOn
+    },
+    {
+      title: t('language'),
+      dataIndex: 'language',
+      render: lang => <CountryFlagAndName code={languageData[lang]?.flag} name={languageData[lang]?.name || lang} />
     }
```

Comment:
Just from consistency perspective, would be nice to go with either defining the render function inline or separately (like `renderUpdateOn`).
For me separately is better, makes reading easier. If you go for this, please also do the same for tags 😬

Reply:
Done, thanks!